core-bundle/src/Routing/ResponseContext/CoreResponseContextFactory.php

```diff
@@ -58,7 +59,10 @@ public function createWebpageResponseContext(): ResponseContext
         $context = $this->createResponseContext();
         $context->add($this->eventDispatcher);
         $context->addLazy(HtmlHeadBag::class);
-        $context->addLazy(JsonLdManager::class);
+
+        $jsonLdManager = new JsonLdManager($context);
+        $jsonLdManager->getGraphForSchema(JsonLdManager::SCHEMA_ORG)->add(new WebPage());
```

Comment:
And now look how easy it would be to add additional data to this type. I'm already loving this new `JsonLdManager` 😊 But let's focus on migrating the existing stuff first.

Reply:
Is there a reason why this does not use `addLazy` with the closure for this?

Reply:
Because the service is built to add the webpage schema?

Reply:
Sorry, I'm sure I'm missing something. Wouldn't the resource only be used when first accessed if done this way:

```php
return (new JsonLdManager($context))
    ->getGraphForSchema(JsonLdManager::SCHEMA_ORG)
    ->add(new WebPage())
;
```
What's the difference to the `HtmlHeadBag` where we're doing this?

Reply:
Yeah we could do that if we think it makes sense (which I don't :D).

Reply:
Still added it in cc21b7203deb5605334c0f9a373ffde510a38822. Your code does not work because it would return the `Graph` instead of the `JsonLdManager` ;)